.github/workflows/test.yml

```diff
@@ -145,7 +140,7 @@ jobs:
           if [[ ${{ matrix.platform.os }} = macos ]]; then
             export PATH="$(brew --prefix llvm@14)/bin:${PATH}"
             export LDFLAGS="-L$(brew --prefix libomp)/lib -L$(brew --prefix llvm@14)/lib -Wl,-rpath,$(brew --prefix llvm@14)/lib"
-            compiler_extra_options="-d:LeopardCmakeFlags='-DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER=$(brew --prefix llvm@14)/bin/clang -DCMAKE_CXX_COMPILER=$(brew --prefix llvm@14)/bin/clang++' -d:LeopardExtraCompilerlags='-fopenmp' -d:LeopardExtraLinkerFlags='-fopenmp -L$(brew --prefix libomp)/lib'"
+            compiler_extra_options="-d:LeopardCmakeFlags='-DENABLE_OPENMP=off -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER=$(brew --prefix llvm@14)/bin/clang -DCMAKE_CXX_COMPILER=$(brew --prefix llvm@14)/bin/clang++' -d:LeopardExtraCompilerlags='-fopenmp' -d:LeopardExtraLinkerFlags='-fopenmp -L$(brew --prefix libomp)/lib'"
```

Comment:
Interesting, why are we disabling opnemp?

Reply:
It was an attempt to get CI to build nim-leopard on macos arm, and disabling openmp was [suggested by csaba](https://github.com/status-im/leopard/pull/1#issuecomment-1146601769) as a way to get this to work.

Reply:
I've since removed this flag, as it didn't work

Reply:
what do you mean by "it didn't work"?
On my system, as of now, leopard is building if I have `-DENABLE_OPENMP=off`.

If instead I have `-DENABLE_OPENMP=on`, it fails to build with this error:
```
[ 77%] Linking CXX executable bench_leopard
ld: Undefined symbols:
  ___kmpc_for_static_fini, referenced from:
      _.omp_outlined. in liblibleopard.a[3](LeopardCommon.cpp.o)
      _.omp_outlined. in liblibleopard.a[4](LeopardFF16.cpp.o)
      _.omp_outlined..2 in liblibleopard.a[4](LeopardFF16.cpp.o)
      _.omp_outlined..4 in liblibleopard.a[4](LeopardFF16.cpp.o)
      _.omp_outlined..6 in liblibleopard.a[4](LeopardFF16.cpp.o)
      _.omp_outlined..8 in liblibleopard.a[4](LeopardFF16.cpp.o)
      _.omp_outlined..10 in liblibleopard.a[4](LeopardFF16.cpp.o)
      ...
```

Maybe, following instructions in https://github.com/codex-storage/nim-leopard#openmp , it does link with OpenMP. Anyway, if we enable that as default, I think it should be a separate PR, and not part of a version bump PR.

Reply:
OK, I see now. We were building with OpenMP in the past in the CI. Then you tried without it, and now you went back to original. All good (if it builds correctly) :-)

Reply:
> OK, I see now. We were building with OpenMP in the past in the CI (on amd64, not on arm64). Then you tried without it, hoping that would fix arm build, but it didn't. And now, you went back to original. All good (if it builds correctly) :-)

Yes. The CI runner changed from amd64 to arm64 (due to using the `macos-latest` runner). So I was trying to get leopard to compile using that runner (arm64). It didn't compile. Then I tried disabling openmp, which didn't help. So then I switched to using the `macos-13` runner which is amd64, and reverted the change to openmp.

> Maybe we would need a separate PR that actually enables arm64 build in the CI,

I agree, and I imagine that eventually the `macos-13` image will reach EOL and we will be forced to use arm64 a *some point*